setup.py

```diff
@@ -25,6 +25,6 @@
     zip_safe=False,
     install_requires=[
         "wagtail>=1.6",
-        "wagtail-bakery>=0.1.0"
+        "wagtail-bakery>=0.3.0"
```

Comment:
Wagtail 2.0 compatibility work has been done at moorinteractive/wagtail-bakery#27 but a new version of the package is still to be released.

This version bump here is just a guess. Once `wagtail-bakery` releases a new version, we need to double check the number and update this accordingly before merging this PR.

Reply:
Version has been released.